tests/integration/ceph_client_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+// Smoke Test for Client CRD
+func (suite *SmokeSuite) TestCreateClient() {
+	logger.Infof("Create Client Smoke Test")
+
+	clientName := "client1"
+	err := suite.helper.UserClient.Create(clientName, suite.namespace)
+	require.Nil(suite.T(), err)
+
+	clientFound := false
+
+	for {
```

Comment:
Could you limit how many times this loop retries? For example, if it takes longer than a minute to create the user, something went wrong and we should fail the test. How about retrying 30 times since we're retrying every 2s?